examples/avro-cli.py

```diff
@@ -36,19 +37,37 @@
 """)
 
 
-# User stores the deserialized user Avro record.
+"""
+    User stores the deserialized user Avro record.
+"""
+
+
 class User(object):
 
-    # Explicitly declare data members to prevent creation of dynamic properties
-    __slots__ = ["name", "favorite_number", "favorite_color"]
+    """
```

Comment:
I believe this will be interpreted as the docstring for the class, not for `__slots__`.
A comment is probably sufficient here.